README.md

```diff
@@ -318,17 +318,19 @@ Tool used to display the defects and fixes in the console output. Currently supp
 
 ### token
 
-Token used to upload findings in SARIF format to GitHub.
+The token is used to upload findings in SARIF format to GitHub.
 
 * default value: `undefined`
 * requirements: `optional`
 
-Token needs to have the following [characteristics](https://docs.github.com/en/rest/code-scanning#upload-an-analysis-as-sarif-data):
+The token needs to have the following [characteristics](https://docs.github.com/en/rest/code-scanning#upload-an-analysis-as-sarif-data):
 
-* Token with the `security_events: write` scope to use this endpoint for private repositories.
-* Token with the `public_repo` scope for **public repositories only**.
+* `security_events: write` scope for **all repositories**.
+* `actions: read` and `contents: read` scopes for **private repositories only**.
```

Comment:
Probably worth phrasing in a way to be clear that the private repo requirements are in-addition-to as opposed to and not instead-of

Reply:
Thank you. I have tried to make it a bit more straightforward. I hope it helps.

Reply:
nice! (it would be nice if github made more of this official in the action.yml format. having a structured format for what permissions are required, under what circumstances, and be consistent between actions would be nice)

Reply:
I agree with this 100%